backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/job/model/PickingJobStepPickedTo.java

```diff
@@ -100,4 +102,12 @@ public List<HuId> getPickedHuIds()
 				.map(HUInfo::getId)
 				.collect(ImmutableList.toImmutableList());
 	}
+
+	@NonNull
+	public Optional<PickingJobStepPickedToHU> getLastPickedHu()
+	{
+		return actualPickedHUs.stream()
+				.min(Comparator.comparing(PickingJobStepPickedToHU::getCreatedAt,
```

Comment:
why not max?

Reply:
(and i think there is another place where I would have the same question... but cannot find it now)